.evergreen/config.in.yml

```diff
@@ -789,6 +789,41 @@ functions:
 
           MONGODB_URI="${MONGODB_URI}" bash ${PROJECT_DIRECTORY}/.evergreen/run-custom-csfle-tests.sh
 
+  "run custom csfle shared lib tests":
+    - command: shell.exec
+      type: test
+      params:
+        silent: true
+        working_dir: "src"
+        script: |
+          ${PREPARE_SHELL}
+          cat <<EOT > prepare_client_encryption.sh
+          export CLIENT_ENCRYPTION=${CLIENT_ENCRYPTION}
+          export CSFLE_KMS_PROVIDERS='${CSFLE_KMS_PROVIDERS}'
```

Comment:
I want to echo Anna's comment here about consistent quote use; on the one hand, it's kind of a nit, but on the other hand, we *should* fix the typo in the readme, and as long as we're pushing changes to this PR, we might as well make the files internally consistent

Reply:
It's all single quoted now.